instrumentation/zipkin.adoc

```diff
@@ -0,0 +1,27 @@
+:imagesdir: ../images
+
+:toc: macro
+:toc-title:
+
+= Zipkin
+
+The Zipkin project contains instrumentation libraries for several programming languages, therefore it makes it easy to get tracing data for polyglot applications. It uses a different approach from the non-intrusive Java agent instrumentation. Developers have to use instrumented libraries or send tracing data directly to the server.  
```

Comment:
Saying "instrumented libraries" implies it is the library that has been instrumented, not that it is a library for instrumenting the application.